src/scenario-context/ScenarioContext.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
```

Comment:
This is the meat of this commit.  With this context and dispatcher in place, we can now access and switch the scenario being viewed globally